Streetcode/Streetcode.DAL/Entities/News/News.cs

```diff
@@ -20,8 +20,8 @@ public class News
         [Required]
         [MaxLength(100)]
         public string URL { get; set; }
-        public int? ImageId { get; set; }
-        public Image? Image { get; set; }
+        public int ImageId { get; set; }
+        public Image Image { get; set; }
```

Comment:
It will create the nullable warning from the linter because you created a not-nullable property but didn't set any value. You can use the nullable version or manually set `default!` or `null!` to `Image`  navigation property.